lighthouse-core/report/styles/report.css

```diff
@@ -29,10 +29,10 @@ span, div, p, section, header, h1, h2, li, ul {
   --text-color: #222;
   --secondary-text-color: #606060;
   --accent-color: #3879d9;
-  --poor-color: #e81e1b;
-  --good-color: #008a07;
-  --average-color: #9f6d00;
-  --warning-color: #947200;
+  --poor-color: #e53935; /* md red 600 */
+  --good-color: #43a047; /* md green 600 */
+  --average-color: #ef6c00; /* md orange 800 */
+  --warning-color: #757575; /* md grey 600 */
```

Comment:
In isolation this looks a bit odd, that a warning icon is done in gray.
I'm presuming you're okay with that because any warning debugString is displayed in red?

Personally I prefer a yellow/orange/mustard color for these error states, but I can live with this just fine. :)